### PR TITLE
quack

### PR DESCRIPTION
**Pull request by PopNotes**
:cl: PopNotes
fix: Duck goes quack. Quack quack quack. Duck goes quack quack quack.
/:cl:

### CODE REVIEW

File: hippiestation.dme [hunk 1]
```diff
@@ -2353,6 +2353,7 @@
 #include "hippiestation\code\game\objects\structures\manned_turret.dm"
 #include "hippiestation\code\game\objects\structures\tables_racks.dm"
 #include "hippiestation\code\game\objects\structures\tank_dispenser.dm"
+#include "hippiestation\code\game\objects\structures\watercloset.dm"
```
Comment (McBawbaggings) [inline hippiestation.dme]: ?

**Comment by banthebantz** [comment]
+1

**Comment by McBawbaggings** [comment]
good

**Comment by PopNotes** [comment]
buon

**Comment by McBawbaggings** [comment]
did you just merge your own code on +2 idiot

**Comment by McBawbaggings** [comment]
It's +3 now but i don't think a single headmin is gonna be like "fuck off with this" regardless but it be +3 due to the increased vote pool

**Pull request merged**